### PR TITLE
add field ttl_status to sensu_check

### PR DESCRIPTION
**Pull request by jasiek191**
# Pull Request Checklist

## Related Issue

Fixes # .

## General

- [ ] Update `README.md` with any necessary configuration snippets

- [ ] New parameters are documented

- [ ] New parameters have tests

- [ ] Tests pass - `bundle exec rake validate lint spec`

### CODE REVIEW

**Comment by ghoneycutt** [comment]
Thanks @jasiek191 for the contribution! Current the spec tests are not passing, could you have a look.

**Comment by jasiek191** [comment]
Hi
I didn't saw it, but now is fixed

File: spec/defines/sensu_check_spec.rb [hunk 1]
```diff
@@ -105,7 +105,8 @@
         :handle              => true,
         :publish             => true,
         :auto_resolve        => true,
-        :ttl                 => 30
+        :ttl                 => 30,
+        :ttl_status          => 1
```
Comment (ghoneycutt) [inline spec/defines/sensu_check_spec.rb]: what happens if ttl_status is a non integer and not `'absent'` ?

Comment (jasiek191) [reply spec/defines/sensu_check_spec.rb]: if is set to absent then have default value 1 (which mean warning)
I don't know what happens if will be set to string, probably sensu client die.
Probably we should check also ttl, interval refresh etc

Comment (ghoneycutt) [reply spec/defines/sensu_check_spec.rb]: I meant, what will Puppet do? We want some checks to ensure that Puppet errors appropriately.

**Comment by ghoneycutt** [comment]
@treydock could you please look into this. It seems it just needs tests before it can be merged.

**Comment by ghoneycutt** [comment]
merged in PR #1000

**Comment by ghoneycutt** [comment]
Thanks @jasiek191 and @treydock !! This has been released in v2.57.0

**Pull request closed without merging**